sdk/core/Cargo.toml

```diff
@@ -31,14 +31,18 @@ bytes = "1.0"
 hyper-rustls = { version = "0.22", optional = true }
 failure = "0.1"
 async-trait = "0.1"
+futures-io = "0.3"
 oauth2 = "=4.0.0-beta.1"
-reqwest = { version = "0.11", optional = true }
+reqwest = { version = "0.11", features = ["stream"], optional = true }
 paste = "1.0"
 md5 = "0.7"
+rand = "0.8"
+async-std = "1.9"
```

Comment:
I don't think `async-std` should be a core dependency. It is an alternative to `tokio`.

Reply:
Yes you are right. As discussed yesterday I've scrapped the code that required `aync-std` (file based input stream).